src/components/king-queen/AboutContent.tsx

```diff
@@ -0,0 +1,24 @@
+export const AboutContent = () => {
+  return (
+    <div className="space-y-8 max-w-2xl">
+      <article>
```

Comment:
Nevim jestli je tady ten article element pouzity uplne spravne, kdyz beru v potaz to, co pisou na https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article tak to uplne nesplnuje ten popis.

> represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). Examples include: a forum post, a magazine